src/pytest_plugins/execute/eth_config/eth_config.py

```diff
@@ -54,37 +73,62 @@ def pytest_configure(config: pytest.Config) -> None:
     Load the network configuration file and load the specific network to be used for
     the test.
     """
-    network_configs_path = config.getoption("network_config_file", default=None)
+    # get flag values
+    network_name = config.getoption("network")
+    network_configs_path = config.getoption("network_config_file")
+    clients = config.getoption("clients")
+    rpc_endpoint = config.getoption("rpc_endpoint")
+    config.option.majority_clients = []  # List[str]
+
+    # load provided networks file
     if network_configs_path is None:
-        network_configs_path = DEFAULT_NETWORK_CONFIGS_FILE
+        pytest.exit("You must specify a value for the --network-config-file flag.")
     if not network_configs_path.exists():
         pytest.exit(f'Specified networks file "{network_configs_path}" does not exist.')
     try:
         network_configs = NetworkConfigFile.from_yaml(network_configs_path)
     except Exception as e:
         pytest.exit(f"Could not load file {network_configs_path}: {e}")
-    network_name = config.getoption("network")
+
     if network_name not in network_configs.root:
         pytest.exit(
             f'Network "{network_name}" could not be found in file "{network_configs_path}".'
         )
     config.network = network_configs.root[network_name]  # type: ignore
 
+    # prepare majority test mode
+    #   parse clients list
+    clients.replace(" ", "")
+    clients = clients.split(",")
+    for c in clients:
+        if c not in EXECUTION_CLIENTS:
+            pytest.exit(f"Unsupported client was passed: {c}")
+    print(f"Provided client list: {clients}")
```

Comment:
I think this printed info is ok since it lets the user know right away of a misconfiguration.

Only question is whether they should be `logger.info` just for consistency of trying not to use bare `print`.